default/AI/fleet_orders.py

```diff
@@ -0,0 +1,471 @@
+from EnumsAI import AIShipRoleType, AIFleetMissionType
```

Comment:
There are enough similarities in this file to the old, slightly differently named one, that I think it would have made for a much more reliable review if you had saved this last bit of name-changing until after everything else was reviewed and taken care of.